recipes/obake/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set version = "0.1" %}
+
+package:
+  name: obake
+  version: {{ version }}
+
+source:
+  url: https://github.com/bluescarni/obake/archive/v{{ version }}.tar.gz
+  sha256: 946de817940421d8af8c614c289b74dc6e221ea803fc2b3dacc30e63d7dc3246
+
+build:
+  number: 0
+  skip: true   # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - clang      # [win]
+    - ninja      # [win]
+
+outputs:
+  - name: obake
+    requirements:
+      host:
+        - abseil-cpp
+        - tbb-devel
+        - tbb
+        - mppp
+        - boost-cpp
+        - backtrace    # [linux]
+      run:
+        - {{ pin_compatible("mppp") }}
```

Comment:
You can remove this from here as `mppp` has `run_exports`.

Reply:
from obake-devel as well?

Reply:
yes.

Reply:
What is the difference in usage between ``pin_subpackage`` and ``pin_compatible`` in the run_exports section?

Reply:
Fixed it. `pin_subpackage` is for packages built in this recipe. `pin_compatible` is for other packages.